### ISSUE TITLE
ns1_zone: CIDR for Secondaries

### ISSUE THREAD

**Issue by yzguy** [terraform-providers/terraform-provider-ns1#79]
Hi there,

### Terraform Version
Run `terraform -v` to show the version. If you are not running the latest version of Terraform, please upgrade because your issue may have already been fixed.

```
# ./terraform -v
Terraform v0.11.7
```

### Affected Resource(s)
Please list the resources as a list, for example:
- ns1_zone

### Terraform Configuration Files
```hcl
resource "ns1_zone" "public_data_zone" {
  zone = "${var.public_data_ns1_zone}"
  ttl = "${var.dns_ttl}"
  secondaries {
    ip     = "10.10.10.10"
    port   = 53
    notify = true
  }
  secondaries {
    ip     = "10.10.10.20"
    port   = 53
    notify = true
  }
  secondaries {
    ip     = "10.10.20.0/27"
  }
}
```

### Debug Output
```
ns1_zone.public_data_zone: Modifying... (ID: 5a984d9d0c13a400017f22c9)
  secondaries.257793059.ip:         "" => "10.10.20.0/27"
  secondaries.257793059.networks.#: "" => "<computed>"
  secondaries.257793059.notify:     "" => "false"
  secondaries.257793059.port:       "" => "53"
  secondaries.3238851087.ip:        "10.10.10.10" => "10.10.10.10"
  secondaries.3238851087.notify:    "true" => "true"
  secondaries.3238851087.port:      "53" => "53"
  secondaries.4276752053.ip:        "10.10.10.20" => "10.10.10.20"
  secondaries.4276752053.notify:    "true" => "true"
  secondaries.4276752053.port:      "53" => "53"
  secondaries.564337443.ip:         "10.10.20.0/27" => ""
  secondaries.564337443.notify:     "false" => "false"
  secondaries.564337443.port:       "0" => "0"

Error: Error applying plan:

1 error(s) occurred:

* ns1_zone.public_data_zone: 1 error(s) occurred:

* ns1_zone.public_data_zone: POST https://api.nsone.net/v1/zones/sub.domain.net: 400 port cannot be set with a subnet address

Terraform does not automatically rollback in the face of errors.
Instead, your Terraform state file has been partially updated with
any resources that successfully completed. Please address the error
above and apply again to incrementally change your infrastructure.
```

### Expected Behavior

NS1 would have accepted a CIDR as a secondary, per [NS1 Documentation](https://help.ns1.com/hc/en-us/articles/360022246733-Configuring-NS1-as-the-primary-DNS-provider)

> NS1 now allows you to configure inbound zone transfer requests from a subnet if you enter a CIDR notation (ex. 192.0.2.0/24) instead of a single IP address in the address field. Note that doing so automatically disables notification options. In order to notify individual servers within the subnet, enter their IP addresses as additional secondary servers explicitly.

### Actual Behavior

There was an error from the API. The default port for a secondary per the provider is 53, which if not defined in the HCL it will be set and POST'd to the API.

### Factoids
The JSON payload below works to set 2 by IP and 1 by CIDR when POSTing to the API directly

```json
{
  "primary": {
    "enabled": true,
    "secondaries": [
      {
        "ip": "10.10.10.10",
        "notify": true,
        "port": 53
      },
      {
        "ip": "10.10.10.20",
        "notify": true,
        "port": 53
      },
      {
        "ip": "10.10.20.0/27"
      }
    ]
  }
}
```

**Comment by rupa** [comment]
sorry about that, I'll get this fixed asap

**Comment by rupa** [comment]
This should be fixed (#82) and will be in next release. Thanks and apologies again

**Comment by yzguy** [comment]
No sweat! It was fun digging through the provider code to see how it's set up and try to figure out if it was more or something else.

Quick fix is much appreciated.
